src/commons/BrowserUtils.ts

```diff
@@ -917,6 +919,76 @@ export namespace BrowserUtils {
     browser.keys(keysToSend);
   }
 
+  /**
+   * Verify zip content, by inspecting the file names in zip
+   * Sends GET request using provided url, and parse the result as zip file
+   * if expectedNumbOfFiles not provided, will only verify that expected file names are in zip
+   * Otherwise will also validate the size is fit
+   * @param linkToZipFile link to zip file
+   * @param listOfFileNames list of files names expected in zip
+   * @param expectedNumOfFiles (optional) expected number of files in zip
+   */
+  export function verifyFilesInZip(
+    linkToZipFile: string,
+    listOfFileNames: string[],
+    expectedNumOfFiles?: number
+  ): void {
+    Reporter.debug('===Verify zip content===');
+
+    const zipFileNames: string[] = zipToBuffer(linkToZipFile);
+
+    if (
+      expectedNumOfFiles !== undefined &&
+      expectedNumOfFiles !== zipFileNames.length
+    ) {
+      const incorrectLengthErrorMessage: string = `Incorrect number of files. Expected '${JSON.stringify(
+        expectedNumOfFiles
```

Comment:
Why do you need to JSON.stringify a number?

Reply:
removing